src/coreclr/vm/object.h

```diff
@@ -834,6 +834,7 @@ typedef REF<UPTRArray>  UPTRARRAYREF;
 typedef REF<CHARArray>  CHARARRAYREF;
 typedef REF<PTRArray>   PTRARRAYREF;  // Warning: Use PtrArray only for single dimensional arrays, not multidim arrays.
 typedef REF<StringObject> STRINGREF;
+typedef REF<ReflectClassBaseObject> REFLECTCLASSBASEREF;
```

Comment:
This typedef exists in this file around line 1600. It should not be necessary to duplicate it here.